app/models/user.rb

```diff
@@ -199,6 +200,7 @@ class User < ApplicationRecord
 
   after_create_commit :send_welcome_notification, :estimate_default_language
   after_commit :index_to_elasticsearch, on: %i[create update]
+  after_commit :sync_related_elasticsearch_docs, on: %i[create update]
```

Comment:
Does this wind up as synchronous i/o or worker job? Not sure I 100% follow. But either way, will this cause any performance problems on create/update?

Reply:
Pretty sure it's async: it's calling `searchable.index_to_elasticsearch` https://github.com/thepracticaldev/dev.to/blob/708508278179bd780c77c47a8bef775948dab30a/app/models/concerns/searchable.rb#L6-L8